nva-datamodel-java/src/test/java/no/unit/nva/PublicationMapperTest.java

```diff
@@ -81,6 +81,13 @@ public void canMapPublicationAndContextToPublicationResponse() throws Exception
         assertNotNull(response);
         assertThat(response.getId(), notNullValue());
         assertThatIdIsPresent(response);
+        assertThatTypIsPresent(response);
+    }
+
+    private void assertThatTypIsPresent(PublicationResponse response) throws JsonProcessingException {
+        String string = dataModelObjectMapper.writeValueAsString(response);
+        ObjectNode json = (ObjectNode) dataModelObjectMapper.readTree(string);
+        assertThat(json.has("type"), is(equalTo(true)));
```

Comment:
we can avoid the generic assertThat(assertMyExpectation(),is(true)) here by using this:
https://github.com/spotify/java-hamcrest#json-matchers